integrations/cyberchef/compose-tests.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+
+services:
```

Comment:
no need to duplicate code of `compose.yml` here. Thanks to the concept of "override", we can write few lines. See the most recent refactor here https://github.com/intelowlproject/IntelOwl/blob/master/integrations/malware_tools_analyzers/compose-tests.yml